test/ts/test-data/flat.ts

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 interface FlatInterface {
+  id: 5;
```

Comment:
Why were these properties added? I'm happy to keep that addition, just wanted to know the reason.

Reply:
To test the LiteralTypes :)

Reply:
Sounds good to me